lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -459,33 +459,6 @@ def test_show_answer_doesnt_write_to_csm(self):
         csmh = BaseStudentModuleHistory.get_history(student_module)
         self.assertEqual(len(csmh), 3)
 
-    def test_grade_with_max_score_cache(self):
-        """
-        Tests that the max score cache is populated after a grading run
-        and that the results of grading runs before and after the cache
-        warms are the same.
-        """
```

Comment:
Similar question here: are there sufficient block_structure cache tests that we can just remove this?